projects/storefrontlib/shared/services/import-export/export-csv.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { FileOptions } from '../../models/file';
+import { ExportService } from './export.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportCsvService {
```

Comment:
opinionated: We have many abbreviations in our system. What about being more specific that it's a csv _file_? (same for import service). i.e. `ExportCsvFileService`